06-building-a-program-start-to-finish/solving-the-problem/output/invertedindexutil.js

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+/**
+ * @class InvertedIndexUtility
+ */
+class InvertedIndexUtility {
+/**
+   * filterIndexed method takes an already indexed array
+   * and filters it to match the query
+   * @param {Array} query - An array of the words
+   * gotten from the sanitized query
+   * @param {Array} indexedFile - An already indexed file
+   * @return {Array} - The queried indexed array
+   * @memberOf InvertedIndexUtility
+   */
+  static filterIndexed(query, indexedFile) {
```

Comment:
what is "query"? is that supposed to be a parameter name? I believe you can used something that best describes what the parameter is.